keyboards/handwired/chiron/rules.mk

```diff
@@ -0,0 +1,28 @@
+# Required Items
+MCU = atmega32u4
+BOOTLOADER = atmel-dfu
+LAYOUTS = chiron
```

Comment:
This option is only for the [Community Layouts feature](https://docs.qmk.fm/#/feature_layouts). You don't need this rule.

```suggestion
```